sklearn/ensemble/weight_boosting.py

```diff
@@ -308,10 +308,11 @@ class AdaBoostClassifier(BaseWeightBoosting, ClassifierMixin):
 
     Parameters
     ----------
-    base_estimator : object, optional (default=DecisionTreeClassifier)
+    base_estimator : object, optional (default=None)
         The base estimator from which the boosted ensemble is built.
         Support for sample weighting is required, as well as proper `classes_`
-        and `n_classes_` attributes.
+        and `n_classes_` attributes. If we chose the default value,
```

Comment:
(1) ``If we chose the default value`` -> If ``None``
(2) Please use double backticks around DecisionTreeClassifier(max_depth=1)
(3) Please change the single backtick around classes_ and n_classes to double backticks